### DISCUSSION TITLE
Welcome to Cosmos-Server Discussions!

### DISCUSSION THREAD

**Discussion by azukaar** [azukaar/Cosmos-Server#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RetroGaming12221** [comment]
does cosmos support regex?

**Reply by azukaar** [reply]
I haven't implemented REGEX no but that is a great suggestion!
